third-party-libs/index.md

```diff
@@ -1,7 +1,12 @@
-# Module content
+# Third-party libraries
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+## Table of contents
+
+1. Finding libraries
+2. Determining library compatibility
+3. Installing and linking libraries
+   1. Android
+   2. IOS
+4. Adding icons
+5. Google Maps integration
+6. Stripe integration
```

Comment:
Does it make sense to add topic about communication between RN and native modules here?

Reply:
will cover this in [React Native Advanced module](https://github.com/rolling-scopes-school/react-native-course/issues/57)